package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": "12 || 14 || 15 || 16",
+    "node": "^12.13 || 14 || 15 || 16",
```

Comment:
I thought we used just the major versions in order to not have to bump for minor updates?

Reply:
But declaring `12` allows `>= 12.0 && <12.13`, which would conflict with the requirements of the `style-loader`. If we don't adjust Hops, someone who's running a Hops app on Node 12.12 would suddenly get an error during dependency installation after a patch update.